package.json

```diff
@@ -195,5 +195,8 @@
     "webpack-stream": "^5.2.1",
     "webpackbar": "^3.1.5",
     "whatwg-fetch": "^3.0.0"
+  },
+  "dependencies": {
+    "psl": "^1.4.0"
```

Comment:
Let's move this together with the rest for now. Not sure if this has been discussed before so let's follow the current convention and open an issue for updating how things are placed here if it hasn't been decided yet.